tox_travis.py

```diff
@@ -67,10 +68,33 @@ def guess_python_env():
     return 'py{major}{minor}'.format(major=major, minor=minor)
 
 
+def get_default_envlist(version):
+    """Parse a default tox env based on the version.
+
+    The version comes from the ``TRAVIS_PYTHON_VERSION`` environment
+    variable. If that isn't set, then use sys.version_info to come
+    up with a reasonable default.
+    """
+    # If version isn't set, then guess from sys.version_info
+    if not version:
+        return guess_python_env()
+
+    # Assume single digit versions
+    match = re.match(r'^(\d)\.(\d)$', version)
```

Comment:
OK, I just took a look at that PEP to find out about `LooseVersion`, and I don't think it's appropriate here, mainly because I really need to have exactly a major version and a minor version in order to make a `py35` tox-style default env name. I will go ahead and accomodate versions that have a third section, and even allow multiple digits in that case.